src/raty.js

```diff
@@ -713,7 +713,7 @@ class Raty {
   _setTitle(score, evt) {
     if (score) {
       const integer = parseInt(Math.ceil(score), 10);
-      const star = this.stars[integer - 1];
+      const star = this.stars.item(integer - 1);
```

Comment:
Is it to avoid out of range value? I think we have an issue with mouseover when it's close to the cancel button where the value comes as negative, but I think it was already addressed.. don't know.

Reply:
Coz `this.stars` is not an array, but iterable.